org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalReplacementProvider.java

```diff
@@ -239,10 +249,15 @@ private void appendGuessingCompletion(StringBuilder buffer, CompletionProposal p
 			}
 
 			char[] argument = parameterNames[i];
-
-			buffer.append("{{");
+			if(supportSnippets){
+				buffer.append("${");
+				buffer.append(Integer.toString(i+1));
+				buffer.append(":");
+			}
 			buffer.append(argument);
```

Comment:
My concern is until we properly implement #12, user will be displayed all the overloaded methods that will essentially insert the same result. 

Would the signature help be enough for you. Do you already have support for it on Vim client?